src/Avalonia.Base/Threading/DispatcherPriority.cs

```diff
@@ -45,32 +45,37 @@ private DispatcherPriority(int value)
         /// <summary>
         /// The job will be processed after other non-idle operations have completed.
         /// </summary>
-        public static readonly DispatcherPriority Background = new(1);
+        public static readonly DispatcherPriority Background = new(MinValue + 1);
 
         /// <summary>
         /// The job will be processed with the same priority as input.
         /// </summary>
-        public static readonly DispatcherPriority Input = new(2);
+        public static readonly DispatcherPriority Input = new(Background + 1);
 
         /// <summary>
         /// The job will be processed after layout and render but before input.
         /// </summary>
-        public static readonly DispatcherPriority Loaded = new(3);
-        
+        public static readonly DispatcherPriority Loaded = new(Input + 1);
+
         /// <summary>
         /// The job will be processed with the same priority as render.
         /// </summary>
-        public static readonly DispatcherPriority Render = new(5);
-        
+        public static readonly DispatcherPriority Render = new(Loaded + 1);
+
         /// <summary>
         /// The job will be processed with the same priority as composition updates.
         /// </summary>
-        public static readonly DispatcherPriority Composition = new(6);
-        
+        public static readonly DispatcherPriority Composition = new(Render + 1);
+
         /// <summary>
-        /// The job will be processed with the same priority as render.
+        /// The job will be processed with the same priority as composition updates.
```

Comment:
The summary is the same for `Composition` and `PreComposition`. It's not clear what the difference is between these two priorities other than PreComposition is done before Composition. Perhaps clearer comments would help here.